packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visit-detail.component.tsx

```diff
@@ -15,11 +15,12 @@ function formatDateTime(date) {
 interface VisitDetailComponentProps {
   visit: Visit;
   patientUuid: string;
+  toggleAll: boolean;
 }
 
-const VisitDetailComponent: React.FC<VisitDetailComponentProps> = ({ visit, patientUuid }) => {
+const VisitDetailComponent: React.FC<VisitDetailComponentProps> = ({ visit, patientUuid, toggleAll }) => {
   const { t } = useTranslation();
-  const [listView, setListView] = useState(true);
+  const [listView, setListView] = useState(false);
```

Comment:
Why are there two pieces of state for the same thing? Is the control inside or outside this component? This implementation seems like it would produce inconsistent state between the two components, which I would expect to cause bugs.

Reply:
@brandones made adjust to fix this issue

Reply:
Oh man, @donaldkibet , I'm so sorry. I'm re-reading this and realizing that your original implementation was totally correct. This new one is buggy—it looks like the main "toggle all" button will swap the open/closed states of each item, which would be strange. You can just revert that latest change and we'll call this good. My bad.